client/src/utils/prettifyStatus.js

```diff
@@ -5,12 +5,20 @@ import { Button } from '../components/generic';
 
 export const prettifyStatus = (status, id, tabValue, handleEngage) => {
   let firstStatus = status[0];
+  let isWithdrawn = false;
   if (status[0] === 'offer_made') firstStatus = 'Offer Made';
   if (status[0] === 'open') firstStatus = 'Open';
   if (status[0] === 'prospecting') firstStatus = 'Prospecting';
   if (status[0] === 'interviewing') firstStatus = 'Interviewing';
   if (status[0] === 'rejected') firstStatus = 'Archived';
   if (status[0] === 'hired') firstStatus = 'Hired';
+  if (status.includes('withdrawn')) {
```

Comment:
nitpick: is there a reason we're using different logic here than the previous statements?

Reply:
1. all previous one status 0 used as display status (which is actual stored status) but here withdrawn a computed status is used whose position is not fixed.
2. All previous cases we are setting single var firstStatus and a flag